.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Binary Release
+
+on: 
+  release:
+    types: [created]
+
+jobs:
+  releases-matrix:
+    name: Release Go Binary
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # build and publish in parallel: linux/386, linux/amd64, 
+        # windows/386, windows/amd64, darwin/386, darwin/amd64 
+        goos: [linux, windows, darwin]
+        goarch: ["386", amd64]
```

Comment:
do we even need 386?

Reply:
🤷‍♂️ i dont think it matters, i can remove it